model/textparse/openmetricsparse.go

```diff
@@ -94,14 +94,49 @@ type OpenMetricsParser struct {
 	exemplarVal   float64
 	exemplarTs    int64
 	hasExemplarTs bool
+
+	// lines ending with _created are skipped by default as they are
+	// parsed by the CreatedTimestamp method. The skipCT flag is set to
+	// false when the CreatedTimestamp method is called.
+	skipCTSeries bool
+}
+type openMetricsParserOptions struct {
+	// SkipCT skips the parsing of _created lines.
+	SkipCTSeries bool
+}
+
+type OpenMetricsOption func(*openMetricsParserOptions)
+
+func WithOMParserCTSeriesSkipped(skipCTSeries bool) OpenMetricsOption {
+	return func(o *openMetricsParserOptions) {
+		o.SkipCTSeries = skipCTSeries
+	}
 }
 
 // NewOpenMetricsParser returns a new parser of the byte slice.
 func NewOpenMetricsParser(b []byte, st *labels.SymbolTable) Parser {
 	return &OpenMetricsParser{
+		l:            &openMetricsLexer{b: b},
+		builder:      labels.NewScratchBuilderWithSymbolTable(st, 16),
+		skipCTSeries: false,
+	}
+}
+
+// NewOpenMetricsParserWithOpts returns a new parser of the byte slice with options.
+func NewOpenMetricsParserWithOpts(b []byte, st *labels.SymbolTable, opts ...OpenMetricsOption) Parser {
+	parser := &OpenMetricsParser{
 		l:       &openMetricsLexer{b: b},
 		builder: labels.NewScratchBuilderWithSymbolTable(st, 16),
 	}
+	DefaultOpenMetricsParserOptions := openMetricsParserOptions{
+		SkipCTSeries: true,
```

Comment:
I assume we decided false by default? Happy to reconsider, but might require a discussion

Reply:
Also why "public" variable name in function? also why default in name if we change it?

Reply:
addressed this and some other resolved comments in recent commit